boards/arm/96b_carbon/Kconfig.defconfig

```diff
@@ -10,4 +10,47 @@ if BOARD_96B_CARBON
 config BOARD
 	default 96b_carbon
 
+if I2C
+
+config I2C_1
```

Comment:
sure no problem; but I was only following what you and the rest of the Zephyr maintainers have done on the frdm_k64f platform (check frdm_k64f/Kconfig.defconfig). Do you then want to follow a different pattern on the ST platforms? just let me know please